net/microsocks/files/microsocks.init

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh /etc/rc.common
+USE_PROCD=1
+START=95
+STOP=01
+start_service() {
+    procd_open_instance
+    
+    procd_set_param respawn 
+
+    port="$(uci_get "microsocks.microsocks.port")"
```

Comment:
this is not the way to go. please take https://github.com/openwrt/packages/blob/4d667ec8e8ffb23e2d42a42c58f0318b3d248dff/net/xray-core/files/xray.init as an example.

Reply:
OK. I hope recent rework is "way to go".